### PR TITLE
[API] Remove experimental tag from ApiBundle

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13|
| Bug fix?        | no                                                      |
| New feature?    | no                                                      |
| BC breaks?      | no                                                      |
| Deprecations?   | no |
| Related tickets | |
| License         | MIT                                                          |

### CODE REVIEW

File: src/Sylius/Component/Core/Grid/Filter/ResourceAutocompleteFilter.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
 use Sylius\Component\Grid\Filtering\FilterInterface;
 use Webmozart\Assert\Assert;
 
-/** @experimental */
+
```
Comment (TheMilek) [inline src/Sylius/Component/Core/Grid/Filter/ResourceAutocompleteFilter.php]:
```suggestion
```

**Review by TheMilek — DISMISSED** [review]

File: src/Sylius/Bundle/ApiBundle/Command/Account/SendAccountRegistrationEmail.php [hunk 2]
```diff
@@ -13,11 +13,7 @@
 
 namespace Sylius\Bundle\ApiBundle\Command\Account;
 
-/**
- * @experimental
- *
- * @immutable
- */
+/** @immutable g*/
```
Comment (Rafikooo) [inline src/Sylius/Bundle/ApiBundle/Command/Account/SendAccountRegistrationEmail.php]: A typo has crept in

**Review by Rafikooo — COMMENTED** [review]
I believe the `experimental` label in the swagger docs should also be removed in this PR.

**Review by Rafikooo — APPROVED** [review]

**Comment by Rafikooo** [comment]
Thank you, @GSadee!

**Pull request merged**